openapi/schema/schema.go

```diff
@@ -82,3 +84,21 @@ func (s *Schema) IsRequired(property string) bool {
 	}
 	return false
 }
+func (s *Schema) GetName() []string {
```

Comment:
Nice. And I mean it this time

Reply:
The triplicate `if something == nil` checks bother me. I'd have used something like a preprocessor macro here if this were C. :/

But thanks!

Reply:
quack quack